.devcontainer/bazel-base/Dockerfile

```diff
@@ -56,6 +56,7 @@ RUN echo "Install general purpose packages" && \
         netbase \
         python${PYTHON_VERSION} \
         python-is-python3 \
+        python3-distutils \
```

Comment:
Is actually installed as a dependency of one of the folly dependencies in line 73. This lib is needed for bazel `pip_parse` running correctly. Added here as a precaution because it is planned to remove folly at some point.

Reply:
```suggestion
        python3-distutils `#  needed for bazel pip_parse running correctly` \
```

Reply:
nice, didn't know that you can put comments like this after items. updated this also for the other entries - and removed some that were outdated/obvious out of date.